apis/v1alpha1/generator.yaml

```diff
@@ -209,6 +209,7 @@ resources:
           path: SourceSnapshotName
       ClusterName:
         is_primary_key: false
+        is_required: false
```

Comment:
I don't see any changes to the API file which I would expect to see if this worked

Reply:
Hopefully it could work

Reply:
I think I understand - the `v0.23.1` code-generator fixed a bug where we weren't properly setting `required` on fields that were imported as custom. Because we now are, and this field was required, it was marking it as such even though we don't want it to be. So setting the `is_required` manually suppresses that. 👍🏼